python/cudf/cudf/core/dataframe.py

```diff
@@ -3902,6 +3902,39 @@ def repeat(self, repeats, axis=None):
         result = DataFrame(data=dict(zip(column_names, cols)))
         return result.set_index(new_index)
 
+    def tile(self, reps):
```

Comment:
Need docstring for this

Reply:
fwiw, I was a little confused with the original description...

> Adds cudf::tile which is a utility to tile a cudf::table. Meaning, the dataframe is repeated along the column axis, the specified number of times.

... because it sounds similar to `repeat` ? But when describing `repeat`, I'd use the word "row" and not "column", so I don't actually have a suggestion here. It's possible I'm just not used to the terminology. :man_shrugging:

Reply:
@cwharris I had a really hard time trying to write the libcudf docxygen for this. How to differentiate it from repeat. I ended up just using Ann example.